andela/mrm_push
# #164858840 List edit and delete subscribers

#### What does this PR do?
Allow users to list, edit and delete push notification subscribers
#### Description of Task to be completed?
Currently, the push notification service has an interface that can be used to add subscribers. This PR extends this UI by allowing for simple listing, editing and deleting subscribers. On listing subscribers, provide a link to their notifications received from the calendar API. 

#### How should this be manually tested?
In order to test it locally:
- clone the branch `ft-create-push-notification-log-164722412`
- Create the channel and the subscribers 
- Hit the `/notifications` endpoint and you should be able to view the different subscribers
- If you navigate to the `push notification link` you should be able to view the individual push notifications
- If you click on either `delete` or `edit`. You should be able to either delete or edit

- [x] add an endpoint to get all subscribers

- [x] add an endpoint to delete a subscriber


- [x] add an enpoint to edit subscriber info

- [x] add UI for `edit` and `delete` subscriber

- [x] add an extension of the UI template for `get all subscribers`


#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#164858840](https://www.pivotaltracker.com/story/show/164858840)
